cmd/influxd/upgrade/upgrade.go

```diff
@@ -115,13 +115,13 @@ func NewCommand() *cobra.Command {
 		Use:   "upgrade",
 		Short: "Upgrade a 1.x version of InfluxDB",
 		Long: `
-    Upgrades a 1.x version of InfluxDB by performing following actions:
+    Upgrades a 1.x version of InfluxDB by performing the following actions:
       1. Reads 1.x config file and creates 2.x config file with matching options. Unsupported 1.x options are reported.
       2. Copies and upgrades 1.x database files.
```

Comment:
```suggestion
      2. Copies 1.x database files.
```

It doesn't make any changes to the files, so no need to suggest they are upgraded